### ISSUE TITLE
ARM build for webkit is disabled

### ISSUE THREAD

**Issue by johanvos** [javafxports/openjdk-jfx#454]
In build.gradle, there is now a hard failure when building webkit for arm. The arm-specific qmake arguments need to be added.

**Comment by kevinrushforth** [comment]
> The arm-specific qmake arguments

I think you mean Cmake (WebKit moved from Qmake to Cmake quite a while ago).

**Comment by johanvos** [comment]
You're right. The build rules for the default platforms has been converted from Qmake to Cmake, but the ARM build logic is not yet converted.
